src/components/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';  
+
+export default function Navbar() {
+    return (
+        <nav className="topbar">
+            <div class="logoDiv">
+                <img src="../images/navbar-logo.png" alt="logo"/>
```

Comment:
It is not getting loaded because of the wrong path.
You can import it first and then add it as a src.
`import AppLogo from "./images/navbar-logo.png";`